tools/setup_env.sh

```diff
@@ -35,7 +35,7 @@ setup_config_file() {
     echo "# Algorand installation path" > $ZIGGURAT_ALGORAND_SETUP_CFG_FILE
     echo "path = \"$ALGORAND_BIN_PATH\"" >> $ZIGGURAT_ALGORAND_SETUP_CFG_FILE
     echo "# Start command with possible arguments" >> $ZIGGURAT_ALGORAND_SETUP_CFG_FILE
-    echo "start_command = \"$ALGOD_BIN_NAME\"" >> $ZIGGURAT_ALGORAND_SETUP_CFG_FILE
+    echo "start_command = \"./$ALGOD_BIN_NAME\"" >> $ZIGGURAT_ALGORAND_SETUP_CFG_FILE
```

Comment:
Right, that happens when one builds from a source code...

What if we remove the following line and merge path 'path' with 'start_command' before:
https://github.com/runziggurat/algorand/blob/main/src/setup/node.rs#L157

And then we always use the full path for the command